src/DataTypes/Course.jl

```diff
@@ -41,10 +41,13 @@ mutable struct Course
     metrics::Dict{String, Any}          # Course-related metrics
     metadata::Dict{String, Any}         # Course-related metadata
 
+    termReq::Int                        # Number of terms that must be completed before enrolling
+    passrate::Float64                   # Percentage of students that pass the course
+
     # Constructor
     function Course(name::AbstractString, credit_hours::Real; prefix::AbstractString="", learning_outcomes::Array{LearningOutcome}=Array{LearningOutcome,1}(),
                     num::AbstractString="", institution::AbstractString="", college::AbstractString="", department::AbstractString="",
-                    cross_listed::Array{Course}=Array{Course,1}(), canonical_name::AbstractString="", id::Int=0)
+                    cross_listed::Array{Course}=Array{Course,1}(), canonical_name::AbstractString="", id::Int=0, termReq::Number=0, passrate::Float64=0.5)
```

Comment:
Since termReq is defined as an Int I think it would make sense to set the default as that same concrete type instead of Number?

Reply:
termReq is removed.